companion/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "disguise",
+	"name": "disguise",
+	"shortname": "d3-osc",
+	"description": "OSC plugin for Companion",
+	"version": "3.0.0",
+	"license": "MIT",
+	"repository": "git+https://github.com/bitfocus/companion-module-disguise.git",
+	"bugs": "https://github.com/bitfocus/companion-module-disguise/issues",
+	"maintainers": [
+		{
+			"name": "Dan Hall",
+			"email": "dan@qed-productions.com"
+		}
+	],
+	"legacyIds": ["disguise_osc", "osc"],
```

Comment:
Where were these ID's previously used? I did a quick check of past commits and didn't see them being used, but I only did a brief check.

Reply:
err, yeah, more inadvertant newb mess on my part. I've removed that line completely